src/modules/nmx/NMXInstrument.cpp

```diff
@@ -167,7 +165,7 @@ void NMXInstrument::processReadouts(void) {
     XTRACE(DATA, DEB, "Plane %u, Coord %u, Channel %u, Panel %u", Plane, Coord,
            readout.Channel, Panel);
     builders[Panel].insert({TimeNS, Coord, ADC, Plane});
-    XTRACE(DATA, DEB, "inserted to builder");
+    //XTRACE(DATA, DEB, "inserted to builder");
```

Comment:
Why do you comment this? This is already optional DEBUG output. If it's obsolete just delete it.

Reply:
Fair enough. It was added by Jenny and was annoying me ;-)